Sources/llbuild2fx/KeyConfiguration.swift

```diff
@@ -43,14 +43,14 @@ public final class KeyConfiguration<K: FXVersioning>: Encodable {
     }
 
     static func resolveKeys(_ keys: [ConfigurationKey], universe: [String]) -> FXSortedSet<String> {
-        return FXSortedSet(keys.flatMap { key in
+        return FXSortedSet(keys.reduce([String](), { (result, key) in
```

Comment:
@swsnider this results in quadratic behaviour because on every step (/O(n)/) we need to create a fresh Array (/O(n)/) which becomes /O(n^2)/ on aggregate.

Please use `reduce(into: []) { ...` and cann `result.append(val)`/`result.append(contentsOf: universe.filter(...)` and that then becomes (amortised) /O(n)/ because only one array will ever be created.